src/cases/components/case-hearings-list/case-hearings-list.component.ts

```diff
@@ -50,6 +52,12 @@ export class CaseHearingsListComponent implements OnInit {
     }
   }
 
+  public ngOnDestroy(): void {
+    if (this.sub) {
+      this.sub.unsubscribe();
+    }
+  }
```

Comment:
is this sub still needed?

Reply:
I have removed this code which is not required.